updateHostsFile.py

```diff
@@ -1312,6 +1316,7 @@ def flush_dns_cache():
                         print_failure(service_msg.format(result="failed"))
                     else:
                         print_success(service_msg.format(result="succeeded"))
+                    restarted_services += [service]
```

Comment:
Please prefer `restarted_services.append()` as used in the rest of the script.

It's not for us, it for future Python contributors.

Reply:
Understandable, done.